File: backend/LeitnerPlatform.Core/Entities/Announcement.cs
using System;

namespace LeitnerPlatform.Core.Entities
{
    public class Announcement
    {
        public Guid Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Content { get; set; } = string.Empty;
        public DateTime PublishedAt { get; set; }
    }
}

File: mobile-app/lib/features/notifications/domain/entities/announcement.dart
import 'package:equatable/equatable.dart';

class Announcement extends Equatable {
  final String id;
  final String title;
  final String content;
  final DateTime publishedAt;

  const Announcement({
    required this.id,
    required this.title,
    required this.content,
    required this.publishedAt,
  });

  @override
  List<Object?> get props => [id, title, content, publishedAt];
}
